.travis.yml

```diff
@@ -14,7 +14,7 @@ env:
   global:
     - _C=build
     - WORKDIR=/spelunky
-    - BUILD_IMAGE=bedzior/spelunky-psp
+    - BUILD_IMAGE=bedzior/spelunky-psp:sdl2
```

Comment:
This could be changed back to implicit `:latest` when confirmed and tagged in Docker Hub

Reply:
So is this going to be changed?

Reply:
I'll do this later and then this PR will be fully ready to be merged

Reply:
Done